service.go

```diff
@@ -310,7 +310,15 @@ func (n *NATSJSEventSource) StartEventSource(src *proto.EventSource, svr proto.E
 		}
 	}
 
-	consumerCtx, err := consumer.Consume(handler)
+	errChan := make(chan error, 1)
+	errHandler := func(consumeCtx jetstream.ConsumeContext, err error) {
+		if err == jetstream.ErrConsumerDeleted || err == jetstream.ErrStreamNotFound {
```

Comment:
This should use `errors.Is()` to make sure it will still work even if a wrapped error is returned.